docker-compose.spark.yml

```diff
@@ -10,12 +10,16 @@ services:
       - "7077:7077"
       - "8888:8888"
     environment:
+      AWS_PROFILE: "raster-foundry"
       SPARK_LOCAL_DIRS: "/tmp"
       SPARK_PUBLIC_DNS: "localhost"
       SPARK_MASTER_WEBUI_PORT: "8888"
       SPARK_DAEMON_MEMORY: "256m"
       SPARK_DAEMON_JAVA_OPTS: "-Dspark.deploy.recoveryMode=FILESYSTEM -Dspark.deploy.recoveryDirectory=/spark-state"
     volumes:
+      - ./app-backend/ingest/src/test/resources:/opt/rf/test-resources
```

Comment:
These volumes _may_ need to be mounted for all workers for the ingest to work on the actual sample spark cluster